### ISSUE TITLE
CLI parameters

### ISSUE THREAD

**Issue by EyalLavi** [ebu/benchmarkstt#44]
To get the conversation going, something like: 
`benchmarkstt --ref reference.txt --hyp hypothesis.txt --metric wer --lan en-GB`

**Comment by MikeSmithEU** [comment]
I propose full verbosity for the long options, with a short option alternative available as well. This allows both shorter commands and full verbose descriptive commands, no "in-between" solution is needed imho.

I.e. 
`benchmarkstt --reference reference.txt --hypothesis hypothesis.txt --metric wer --language en-GB`

and with short options:

`benchmarkstt -r reference.txt -h hypothesis.txt -m wer -l en-GB`

This is in line with how some arguments for normalization is currently implemented, eg. for inputfile:

```
    -i file, --inputfile file
                        read input from this file, defaults to STDIN
```

**Comment by EyalLavi** [comment]
The minimal set of parameters has been implemented in #48 so the proposal is to close this issue and open separate issues for specific additional parameters as development progresses.

**Comment by MikeSmithEU** [comment]
> The minimal set of parameters has been implemented in #48 so the proposal is to close this issue and open separate issues for specific additional parameters as development progresses.

Agreed. Point for discussion though: at the moment we have different 'subcommands' available like `normalization`, `api` and `metrics`. The entire flow would be its own 'subcommand', possibly set as default (i.e. if no subcommand given, do the entire flow).

**Comment by EyalLavi** [comment]
Decided in meeting today to close this ticket.
